ReactNativeClient/lib/SyncTargetAmazonS3.js

```diff
@@ -4,7 +4,7 @@ const Setting = require('lib/models/Setting').default;
 const { FileApi } = require('lib/file-api.js');
 const Synchronizer = require('lib/Synchronizer').default;
 const { FileApiDriverAmazonS3 } = require('lib/file-api-driver-amazon-s3.js');
-const S3 = require('aws-sdk/clients/s3');
+const AWS = require('aws-sdk');
```

Comment:
Please only import what you need and not the whole AWS package. The AWS package is no less than 46 MB, by far the largest package we have even though only a tiny minority of users use AWS sync. So it's best if we only import what we use, so that hopefully some day we can prune the parts that we don't.

Reply:
Thank you, I have made the changes.
Turns out you can also pass the URL directly to the S3 Client.